src/main/java/com/commercetools/sync/commons/models/NonResolvedReferencesCustomObject.java

```diff
@@ -0,0 +1,56 @@
+package com.commercetools.sync.commons.models;
+
+import io.sphere.sdk.products.ProductDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+public class NonResolvedReferencesCustomObject {
```

Comment:
also `final` class if we know that we won't extend it.

Reply:
b4ecc8c49c15182fea9dffd364df66f172dbf6a1